data-types.md

```diff
@@ -3,4 +3,408 @@ title: Data Types
 toc: false
 ---
 
-Coming soon.
\ No newline at end of file
+CockroachDB supports the followinig data types. Click a type for more details.
+
+Type | Description | Example
+-----|-------------|--------
+[`INT`](#int) | A 64-bit signed integer. | `12345`
+[`DECIMAL`](#decimal) | An exact, fixed-point number. | `DECIMAL '1.2345'`
+[`FLOAT`](#float) | An inexact, floating-point number. | `1.2345`
+[`BOOL`](#bool) | A Boolean value. | `true` 
+[`DATE`](#date) | Year, month, day. | `DATE '2016-01-25'`
+[`TIMESTAMP`](#timestamp) | A date and time pairing. | `TIMESTAMP '2016-01-25 10:10:10'`
+[`INTERVAL`](#interval) | A span of time. | `INTERVAL '2h30m30s'`
+[`STRING`](#string) | A string of characters. | `'a1b2c3'`
+[`BYTES`](#bytes) | A string of binary characters. | `'a1b2c3'`
+
+## `INT`
+
+The `INT` type stores 64-bit signed integers, that is, whole numbers from 9,223,372,036,854,775,808 to 9,223,372,036,854,775,807. 
+
+#### Synonyms 
+
+In CockroachDB, the following are synonyms of `INT` and are implemented identically: 
+
+- `SMALLINT` 
+- `INTEGER` 
+- `INT64` 
+- `BIGINT`
+- `BITS`
+
+#### Examples
+
+~~~
+CREATE TABLE ints (a INT PRIMARY KEY, b SMALLINT, c INTEGER);
+
+SHOW COLUMNS FROM ints;
++-------+------+-------+---------+
+| Field | Type | Null  | Default |
++-------+------+-------+---------+
+| a     | INT  | false | NULL    |
+| b     | INT  | true  | NULL    |
+| c     | INT  | true  | NULL    |
++-------+------+-------+---------+
+
+INSERT INTO ints VALUES (11111, 22222, 33333);
+
+SELECT * FROM ints;
++-------+-------+-------+
+|   a   |   b   |   c   |
++-------+-------+-------+
+| 11111 | 22222 | 33333 |
++-------+-------+-------+
+~~~
+
+## `DECIMAL`
+
+The `DECIMAL` type stores exact, fixed-point numbers with no limit on digits to the left and right of the decimal point. This type is used when it is important to preserve exact precision, for example, with monetary data. 
+
+#### Synonyms
+
+In CockroachDB, the following are synonyms of `DECIMAL` and are implemented identically:
+
+- `DEC` 
+- `NUMERIC` 
+
+#### Precision and Scale
+
+Precision (the maximum count of digits in a whole number, both to the left and right of the decimal point) and scale (the maximum count of digits to the right of the decimal point) are always unlimited for a `DECIMAL` value. If you specify these limits by declaring `DECIMAL(precision, scale)` on a column, they will not be enforced. 
+
+#### Format
+
+When declaring a decimal, format it as `DECIMAL '1.2345'`. This casts the value as a string, which preserves the number's exact precision.
+
+Alternately, you can cast a `FLOAT` as a `DECIMAL`, but note that precision will be limited to 17 digits in total (both to the left and right of the decimal point):
+
+- `1.2345::DECIMAL`
+- `CAST (1.2345 AS DECIMAL)`
```

Comment:
OK, I'lll remove the `X::Y` syntax and mention it when I get to the `CAST` function. Fixed with next commit.